### ISSUE TITLE
Can't get Application Name using Deploy with CodeDeploy task after 1.20 extension upgrade

### ISSUE THREAD

**Issue by kramceel** [aws/aws-vsts-tools#250]
**Describe the bug**
In Azure devops pipeline build, we've had a working build step to deploy to AWS Codedeploy but now the step is failing with Can't find Application, which is the application defined in AWS Codedeploy.  This was working until this morning...when the 1.20 version of the extension was upgraded.

**To reproduce**
Add the AWS tools extension to an azure devops build and add the Deploy With CodeDeploy step, then fill out all of the fields.  

**Expected behavior**
This should run correctly

**Screenshots**

BAD
![image](https://user-images.githubusercontent.com/51761517/59380268-924b2400-8d0d-11e9-999d-3cac55a49d24.png)
GOOD
![image](https://user-images.githubusercontent.com/51761517/59381012-51540f00-8d0f-11e9-8fa9-85d930c5b167.png)

**Your Environment**
 - Azure Devops  
 - Azure Devops version: ?
 - AWS VSTS Tools version: 1.20

**Additional context**

**Comment by kaurranjeet12** [comment]
I am seeing same problem. code deploy using this extension is failing since version 1.2 release. It is no longer able to find the application. It was working with version 1.1.8. 

Working here:

```
2019-06-12T14:41:22.6966729Z ##[section]Starting: Deploy with CodeDeploy: webapps-dev
2019-06-12T14:41:22.7074233Z ==============================================================================
2019-06-12T14:41:22.7074343Z Task         : AWS CodeDeploy Application Deployment
2019-06-12T14:41:22.7074475Z Description  : Deploys an application to Amazon EC2 instance(s) using AWS CodeDeploy
2019-06-12T14:41:22.7074553Z Version      : 1.1.8
2019-06-12T14:41:22.7074888Z Author       : Amazon Web Services
2019-06-12T14:41:22.7075062Z Help         : Please refer to [AWS CodeDeploy User Guide](https://docs.aws.amazon.com/codedeploy/latest/userguide/) for more information on working with AWS CodeDeploy.


More information on this task can be found in the [task reference](https://docs.aws.amazon.com/vsts/latest/userguide/codedeploy-deployment.html).


####Task Permissions
This task requires permissions to call the following AWS service APIs (depending on selected task options, not all APIs may be used):
* codedeploy:GetApplication
* codedeploy:GetDeploymentGroup
* codedeploy:CreateDeployment
* codedeploy:GetDeployment


Depending on selected parameters the task may also require permissions to verify your deployment bundle exists in S3 or upload your application bundle to the specified Amazon S3 bucket. Depending on the size of the application bundle, either PutObject or the S3 multi-part upload APIs may be used.
2019-06-12T14:41:22.7075296Z ==============================================================================
2019-06-12T14:41:23.2104384Z Configuring credentials for task
2019-06-12T14:41:23.2112959Z 90686b9a-abe4-46c3-859c-31b5805ae83c exists true
2019-06-12T14:41:23.2114002Z ...configuring AWS credentials from service endpoint '90686b9a-abe4-46c3-859c-31b5805ae83c'
2019-06-12T14:41:23.2114305Z ...endpoint defines standard access/secret key credentials
2019-06-12T14:41:23.2131268Z Configuring region for task
2019-06-12T14:41:23.2133618Z ...configured to use region us-gov-west-1, defined in task.
2019-06-12T14:41:23.2231452Z Configuring credentials for task
2019-06-12T14:41:23.2235423Z 90686b9a-abe4-46c3-859c-31b5805ae83c exists true
2019-06-12T14:41:23.2236766Z ...configuring AWS credentials from service endpoint '90686b9a-abe4-46c3-859c-31b5805ae83c'
2019-06-12T14:41:23.2237692Z ...endpoint defines standard access/secret key credentials
2019-06-12T14:41:23.2238902Z Configuring region for task
2019-06-12T14:41:23.2240854Z ...configured to use region us-gov-west-1, defined in task.
2019-06-12T14:41:24.0610670Z Creating deployment bundle archive from build output at D:\a\r1\a\CI_Bld\REF
2019-06-12T14:41:29.7471844Z Archive size: 50891388 total bytes
2019-06-12T14:41:29.7481387Z Created deployment bundle D:\a\_temp\webapps-dev.v1560350484047.zip
2019-06-12T14:41:29.7482183Z Uploading revision bundle D:\a\_temp\webapps-dev.v1560350484047.zip to object REF/webapps-dev.v1560350484047.zip in bucket deployment-pkgs
2019-06-12T14:41:34.7812161Z Bundle upload completed successfully
2019-06-12T14:41:34.7812400Z Deleting uploaded bundle D:\a\_temp\webapps-dev.v1560350484047.zip created by task
2019-06-12T14:41:34.7984904Z Deploying the new revision
2019-06-12T14:41:35.4632300Z Started deployment of new revision to deployment group ref-app-bluegreen-deploymentgroup for application webapps-dev, deployment ID d-KB1ZQ43JZ
2019-06-12T14:41:35.4637409Z Waiting for deployment to complete
2019-06-12T14:54:23.8518509Z Deployment transitioned to successful status
2019-06-12T14:54:23.8666808Z ##[section]Finishing: Deploy with CodeDeploy: webapps-dev
```



Failed here:

```
2019-06-12T18:36:48.0173988Z ##[section]Starting: Deploy with CodeDeploy: webapps-dev
2019-06-12T18:36:48.0305343Z ==============================================================================
2019-06-12T18:36:48.0305437Z Task         : AWS CodeDeploy Application Deployment
2019-06-12T18:36:48.0305517Z Description  : Deploys an application to Amazon EC2 instance(s) using AWS CodeDeploy
2019-06-12T18:36:48.0305572Z Version      : 1.2.0
2019-06-12T18:36:48.0305654Z Author       : Amazon Web Services
2019-06-12T18:36:48.0305793Z Help         : Please refer to [AWS CodeDeploy User Guide](https://docs.aws.amazon.com/codedeploy/latest/userguide/) for more information on working with AWS CodeDeploy.


More information on this task can be found in the [task reference](https://docs.aws.amazon.com/vsts/latest/userguide/codedeploy-deployment.html).


####Task Permissions
This task requires permissions to call the following AWS service APIs (depending on selected task options, not all APIs may be used):
* codedeploy:GetApplication
* codedeploy:GetDeploymentGroup
* codedeploy:CreateDeployment
* codedeploy:GetDeployment


Depending on selected parameters the task may also require permissions to verify your deployment bundle exists in S3 or upload your application bundle to the specified Amazon S3 bucket. Depending on the size of the application bundle, either PutObject or the S3 multi-part upload APIs may be used.
2019-06-12T18:36:48.0306005Z ==============================================================================
2019-06-12T18:36:48.7103189Z Configuring credentials for task
2019-06-12T18:36:48.7109519Z 90686b9a-abe4-46c3-859c-31b5805ae83c exists true
2019-06-12T18:36:48.7110030Z ...configuring AWS credentials from service endpoint '90686b9a-abe4-46c3-859c-31b5805ae83c'
2019-06-12T18:36:48.7110248Z ...endpoint defines standard access/secret key credentials
2019-06-12T18:36:48.7129621Z Configuring region for task
2019-06-12T18:36:48.7131697Z ...configured to use region us-gov-west-1, defined in task.
2019-06-12T18:36:48.7268726Z Configuring credentials for task
2019-06-12T18:36:48.7271753Z 90686b9a-abe4-46c3-859c-31b5805ae83c exists true
2019-06-12T18:36:48.7272135Z ...configuring AWS credentials from service endpoint '90686b9a-abe4-46c3-859c-31b5805ae83c'
2019-06-12T18:36:48.7272372Z ...endpoint defines standard access/secret key credentials
2019-06-12T18:36:48.7273724Z Configuring region for task
2019-06-12T18:36:48.7275534Z ...configured to use region us-gov-west-1, defined in task.
2019-06-12T18:36:48.7734824Z ##[error]Error: Application webapps-dev does not exist
2019-06-12T18:36:48.7779070Z ##[section]Finishing: Deploy with CodeDeploy: webapps-dev
```

**Comment by darshan-emailage** [comment]
I'm getting the same error

**Comment by sharkguto** [comment]
same error here

**Comment by hunterwerlla** [comment]
We have a hotfix to fix this issue: https://github.com/aws/aws-vsts-tools/pull/252 we will be rolling it out as fast as possible

**Comment by kiiadi** [comment]
Dupe of #246

**Comment by hunterwerlla** [comment]
@kramceel We just released version 1.3.0, can you verify that it fixes your issue?

**Comment by kramceel** [comment]
It worked!  Thank you.
